_data/compatibility/onekey.yaml

```diff
@@ -0,0 +1,61 @@
+---
+name: OneKey
+internal_url: /en/compatibility/onekey
+logo: /img/compatibility/onekey/onekey.png
+rbf:
+  tested:
+    date: "2023-04-07"
+    platforms:
+      - macOS
+    version: "n/a"
+  features:
+    receive:
+      notification: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "false"
+      shows_original_version: "false"
+    send:
+      signals_bip125: "false"
+      list: "untested"
+      details: "untested"
+      shows_replaced_version: "untested"
+      shows_original_version: "untested"
+  examples:
+    - image: /img/compatibility/onekey/rbf/send-screen-1.png
+      caption: >
+        Sending Transaction - Default send (withdrawal) screen. No RBF options.
+    - image: /img/compatibility/onekey/rbf/send-screen-2.png
+      caption: >
+        Sending Transaction - Default send (withdrawal) screen. No RBF options.
+    - image: /img/compatibility/onekey/rbf/send-screen-3.png
+      caption: >
+        Sending Transaction - Default send (withdrawal) screen. No RBF options.
+    - image: /img/compatibility/onekey/rbf/history.png
+      caption: >
+        Sending Transaction - Default send (withdrawal) screen. No RBF options.
+    - image: /img/compatibility/onekey/rbf/receive.png
+      caption: >
+        Attempting Transaction Replacement - Transaction not sent via RBF. No fee bump options.
+
+segwit:
+  tested:
+    date: "2023-04-07"
+    platforms:
+      - macOS
+    version: "4.2.1"
+  features:
+    receive:
+      p2sh_wrapped: "true"
+      bech32: "true"
+      default: "p2sh_wrapped"
+    send:
+      bech32: "true"
+      change_bech32: "true"
+      segwit_v1: "Fails on validation of the address."
```

Comment:
@loatheb does onekey allow sending to taproot addresses?

Reply:
Sorry I missed this part, I just tried OneKey to support it.

https://btc.com/5859760124a47c529c8128e35675f44efeef69f9841aa1a04bf9d759c641f3d7

<img width="1452" alt="image" src="https://user-images.githubusercontent.com/18140164/233523565-82653367-0821-481a-9602-421da682f88c.png">